R/object_name_linters.R

```diff
@@ -49,7 +49,8 @@ object_name_linter <- function(styles = c("snake_case", "symbols")) {
 
     # Retrieve assigned name
     nms <- strip_names(
-      as.character(xml2::xml_find_first(assignments, "./text()")))
+      xml2::xml_text(xml2::xml_find_first(assignments, "./text()"))
```

Comment:
"%<-%" was turned into "%&lt;-%" by `as.character()`

Reply:
those look the same :thinking: anyway, yes the right fix

Reply:
Funny how GitHub unescaped the XML entity. as.character had a AND lt SEMICOLON in its output.

Reply:
ah right. i saw that & was on the XML tree & was puzzled for a minute and moved on. glad we have better test cases for this now.